server/app/views/applicant/ProgramCardsSectionParamsFactory.java

```diff
@@ -195,6 +213,10 @@ public abstract static class ProgramCardParams {
 
     public abstract Optional<String> applicationStatus();
 
+    public abstract String imageSource();
+
+    public abstract String altText();
+
```

Comment:
(blocking) Please make the `imageSource` and `altText` to be type `Optional<String>`